Makefile

```diff
@@ -678,7 +719,7 @@ lint-packages:
 	RUSTFLAGS=${RUSTFLAGS} cargo clippy -p wasmer-compiler
 	RUSTFLAGS=${RUSTFLAGS} cargo clippy -p wasmer-compiler-cranelift
 	RUSTFLAGS=${RUSTFLAGS} cargo clippy -p wasmer-compiler-singlepass
-	RUSTFLAGS=${RUSTFLAGS} cargo clippy -p wasmer-cli
+	RUSTFLAGS=${RUSTFLAGS} cargo clippy --manifest-path lib/cli/Cargo.toml $(compiler_features)
```

Comment:
Since LLVM is likely to be part of `$(compiler_features)` set, the Lint workflow in the CI must install LLVM now.

Reply:
I see. I guess I'll add that to the CI setup. There actually was [one problem in the code](https://github.com/wasmerio/wasmer/pull/2135/files#diff-08e257cac9c32e9684a7544448cf74045946c2d9f643cd0ea3700d0a31cc7139L151-R151) that would have been caught by linting with all on. Should I additionally keep the "lint with all off"?

Reply:
I reckon we should keep the Lints as they are in `master` for the moment, and open another PR or a new issue to update them so that they include `$(compiler_features)` (which makes sense!).

Reply:
That's not an option, as compiling without any compiler_features will fail. But I guess I could temporarily remove linting wasmer-cli and do another PR? Dangerous...